portfolio/src/main/java/com/google/sps/servlets/MessageFormServlet.java

```diff
@@ -0,0 +1,37 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/message-form-handler")
+public class MessageFormServlet extends HttpServlet {
```

Comment:
Since `week3` branches from `week2`, in Github you should mark this branch as merging into the `week2` branch so that these changes will not show up in the PR :) let me know if you have any questions about that!

![Screen Shot 2021-06-21 at 8 41 16 PM](https://user-images.githubusercontent.com/34181040/122845159-10fdd300-d2d1-11eb-872c-0aeb0dc1a147.jpg)